tests/boardfarm_plugins/boardfarm_prplmesh/tests/client_association_link_metrics.py

```diff
@@ -0,0 +1,64 @@
+###############################################################
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+# SPDX-FileCopyrightText: 2020 the prplMesh contributors (see AUTHORS.md)
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+###############################################################
+
+import time
+
+from .prplmesh_base_test import PrplMeshBaseTest
+from capi import tlv
+from environment import Station
+from opts import debug
+
+
+class ClientAssociationLinkMetrics(PrplMeshBaseTest):
+    ''' This test verifies that a MAUT with an associated STA responds to
+    an Associated STA Link Metrics Query message with an Associated STA Link Metrics
+    Response message containing an Associated STA Link Metrics TLV for the associated STA.'''
+
+    def runTest(self):
+        # Locate test participants in array
+        for dev in self.dev:
+            if dev.controller_entity:
+                controller = dev.controller_entity
+            if dev.agent_entity:
+                agent = dev.agent_entity
```

Comment:
It's really time we refactor this setup and teardown work.

I'll convert this comment to an issue and assign it to @aciliketcap

Reply:
Actually, part of setup is refactored in #1486